ecs-cli/modules/utils/compose/ecs_params_reader.go

```diff
@@ -36,15 +38,21 @@ type EcsTaskDef struct {
 	TaskRoleArn          string        `yaml:"task_role_arn"`
 	ContainerDefinitions ContainerDefs `yaml:"services"`
 	ExecutionRole        string        `yaml:"task_execution_role"`
-	TaskSize             TaskSize      `yaml:"task_size"`           // Needed to run FARGATE tasks
+	TaskSize             TaskSize      `yaml:"task_size"` // Needed to run FARGATE tasks
 }
 
+// ContainerDefs is a map of ContainerDefs within a task definition
 type ContainerDefs map[string]ContainerDef
 
+// ContainerDef specifies settings for a specific container
 type ContainerDef struct {
-	Essential bool `yaml:"essential"`
+	Essential         bool  `yaml:"essential"`
+	CPUShares         int64 `yaml:"cpu_shares"`
```

Comment:
Since the `ContainerDef` struct is meant to hold fields that match what's on `ecs.ContainerDefinition`, I would call this `Cpu` .